### DISCUSSION TITLE
기술 찍먹 리스트

### DISCUSSION THREAD

**Discussion by hoo00nn** [z1zon/z1zon-study#1] [퐁당퐁당]
TBD

**Comment by cyjo9603** [comment]
경량화된 react 직접 만들어보기 (여러 주차)

**Comment by cyjo9603** [comment]
markdown parser 만들기

**Comment by hoo00nn** [comment]
디자인 패턴 구현해보기 (주차별로 2~3개?)
- https://refactoring.guru/ko/design-patterns/typescript
- https://www.youtube.com/watch?v=UgtgJ486CDc&list=PLe6NQuuFBu7H3sFnErshsfgNPE9dOZZrx

**Comment by hoo00nn** [comment]
동시편집 아키텍쳐 설계해보기 (같이 토론하면서 해도 좋을듯? 구현까지 해봐도..)

**Comment by newgardener** [comment]
경량화된 react-query 직접 만들어보기 (최소 2주)

**Comment by syoung125** [comment]
마크다운 또는 그냥 에디터 간단히 만들어보기

**Comment by syoung125** [comment]
사진에 필터 적용하는거 구현해보기 (이미지 처리)

**Comment by syoung125** [comment]
코드 에디터 간단히 만들어보기
(코드작성, 실행 가능한 프로그래머스 같은)

**Comment by newgardener** [comment]
Streaming SSR 직접 구현해보기 (최소 2주)

**Comment by hy57in** [comment]
AWS 자격증 따기

**Reply by hoo00nn** [reply]
오..!! 관심있었는데

**Comment by hy57in** [comment]
공부한 것 아티클로 발행하기

**Comment by hy57in** [comment]
chatGPT 등 AI 이용하여 일상에 도움되는 자동화 툴 만들어보기

**Comment by hy57in** [comment]
슬랙봇 만들기

**Comment by hoo00nn** [comment]
예의바르고 신선한 Crawler 구현해보기(?)

**Comment by syoung125** [comment]
웹에서 프리 드로잉 구현
https://blog.mathpresso.com/%EC%9B%B9%EC%97%90%EC%84%9C-%ED%94%84%EB%A6%AC-%EB%93%9C%EB%A1%9C%EC%9E%89-%EB%B0%91%EB%B0%94%EB%8B%A5%EB%B6%80%ED%84%B0-%EA%B5%AC%ED%98%84%ED%95%98%EA%B8%B0-965a0d2253c1

**Comment by syoung125** [comment]
번들러 만들어보기

**Comment by hy57in** [comment]
[rustlings](https://github.com/rust-lang/rustlings) 문제 풀면서 Rust 맛보기 🦀

**Comment by hoo00nn** [comment]
react 19 변경사항 조사 + 느낀점 작성
